docs/faq/items/file_not_formatted.md

```diff
@@ -0,0 +1,5 @@
+# Why is my file not formatted?
+
+Most likely because it is ignored in `.gitignore` or excluded with configuration. See
```

Comment:
"excluded with configuration" reads wrong to me but I'm not sure what would be a good alternative.

Reply:
Hmm, "excluded with command line options"? I originally put "configuration" to include file options, but I think this would get the point across just as well.

Reply:
Did not change this yet, should we?

Reply:
Eh, it's alright as it is :)